financialaid/serializers.py

```diff
@@ -0,0 +1,62 @@
+"""
+Serializers from financialaid
+"""
+import datetime
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from rest_framework.fields import IntegerField, FloatField, CharField
+
+from courses.models import Program
+from dashboard.models import ProgramEnrollment
+from financialaid.api import determine_tier_program, determine_auto_approval
+from financialaid.models import FinancialAid, FinancialAidStatus
+
+
+class FinancialAidSerializer(serializers.Serializer):
```

Comment:
We originally did, but we also needed to pass in a field (program) that isn't on the model, which lead to a bunch of issues with the ModelSerializer's auto validation and data type checking.  We decided it was more efficient to just change it to a regular serializer instead of trying to debug getting it to work with a ModelSerializer.